cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/Cargo.toml

```diff
@@ -43,6 +45,7 @@ sp-api = { workspace = true }
 sp-block-builder = { workspace = true }
 sp-consensus-aura = { workspace = true }
 sp-core = { workspace = true }
+sp-keyring = { workspace = true }
```

Comment:
is keyring/std feature missing below in std feature definition?
(also in other cargo.toml files?)